protocol/protobuf/message.proto

```diff
@@ -0,0 +1,172 @@
+syntax = "proto3";
```

Comment:
I think we can split this file into logical units like `sync.proto`, `message.proto` etc.